rental.rb

```diff
@@ -0,0 +1,19 @@
+# Create the many-to-many (also has-many-through)
+# relationship between Person and Book
+# using the intermediate class Rental
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    # a rental belongs to a book
+    book.rental << self
+
+    @person = person
+    # a rental belongs to a person
+    person.rental << self
```

Comment:
- [ ] As specified in the [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md), " `Create the has-many side of Person and Rental (a person has many rentals).` ", please kindly update your `Person class`(in the person.rb file, not accessible for this PR since you haven't made any change on it) with a method for adding rentals, here he person itself can be sent as a parameter to create the new rental achieving the 'has-many' association. Something like so: 

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```
Please kindly fix that.
#### Please this change is to be implemented in your person.rb file.